src/components/DynamicCalendar/eventCal/EventCalendar.js

```diff
@@ -165,31 +170,38 @@ export default class EventCalendar extends React.Component {
 
   _onBackButton() {
     this.props.backButton()
+    this.props._setScrollEnabled && this.props._setScrollEnabled(true)
   }
 
   render() {
-    const { width, virtualizedListProps, events, initDate } = this.props
+    const { width, virtualizedListProps, flatListProps, events, initDate } = this.props
+
+    let filteredEvents = _.filter(events, (event) => {
+      const eventStartTime = moment(event.start)
+      return (
+        eventStartTime >= moment(initDate).clone().startOf('day') &&
+        eventStartTime <= moment(initDate).clone().endOf('day')
+      )
+    })
+
+    if(filteredEvents.length === 0) {
+      filteredEvents = [{}]
+    }
```

Comment:
would it be worth encapsulating this into its own function or memoizing the value?  I see we're using the componentWillReceiveProps (which is deprecated, but could be useful in this case to set the filtered value in state prior to render?)